tests/test_database.py

```diff
@@ -0,0 +1,64 @@
+import sys
+sys.path.append("/Users/josefinfundin/Dev/Tattoo-Database")
```

Comment:
Can you put a relative path here? (i.e "..")

Reply:
moved to separate file and made relative